src/containers/FileActionMenu.jsx

```diff
@@ -58,19 +103,28 @@ const FileActionMenu = props => (
 )
 
 const mapStateToProps = (state, ownProps) => ({
-  file: getActionableFile(state),
+  files: getActionableFiles(state),
   actionMenu: state.ui.actionMenu
 })
 
 const mapDispatchToProps = (dispatch, ownProps) => ({
-  onDownload: id => {
+  onDownloadFile: id => {
     const clearUI = () => {
       dispatch(actionMenuLoaded('download'))
       dispatch(hideFileActionMenu())
     }
     dispatch(actionMenuLoading('download'))
     dispatch(downloadFile(id)).then(clearUI).catch(clearUI)
   },
+  onDownloadSelection: () => {
+    const clearUI = () => {
```

Comment:
Have you considered declaring this function once and for all, outside of the body of `onDownloadSelection ()`?